### PR TITLE
Deimos coadd2d debug

### PR DESCRIPTION
**Pull request by ShaneBechtel**
This small PR fixes some bugs that were identified in Pypeit 2DCoadds for combinations of frames that are dithered in central wavelength.

### CODE REVIEW

File: pypeit/specobjs.py [hunk 1]
```diff
@@ -409,14 +409,23 @@ def name_indices(self, name):
...
         return indx
 
-    def slitorder_objid_indices(self, slitorder, objid):
+    def slitorder_objid_indices(self, slitorder, objid, toler=5):
         """
         Return the set of indices matching the input slit/order and the input objid
-        """
+        
+        Args:
+            slitorder:
```
Comment (profxj) [inline pypeit/specobjs.py]: Fill in descriptions for these `Args`,  thx!

**Review by profxj — APPROVED** [review]
minor comments from me.
thx for this!

note -- you have to manage the CHANGES conflict

File: pypeit/specobjs.py [hunk 2]
```diff
@@ -409,14 +409,23 @@ def name_indices(self, name):
...
+
+        Returns: 
+
+        """ # SHANE Fill this in.
+
         if self[0].PYPELINE == 'Echelle':
             indx = (self.ECH_ORDER == slitorder) & (self.ECH_OBJID == objid)
         elif self[0].PYPELINE == 'MultiSlit':
-            indx = (self.SLITID == slitorder) & (self.OBJID == objid)
+            indx = (np.abs(self.SLITID - slitorder) <= toler) & (self.OBJID == objid)
```
Comment (debora-pe) [inline pypeit/specobjs.py:435]: I guess you also want to decide if this `toler` is 5, 3, or a parameter. Maybe add a TODO?

**Review by debora-pe — APPROVED** [review]
Looks good! Only one minor comment.

**Comment by profxj** [comment]
@ShaneBechtel -- merge when you are happy with this

**Review by profxj — CHANGES_REQUESTED** [review]
Items keep getting added to this PR.
Am going to close and have it be reopened when ready.

**Comment by profxj** [comment]
Am closing until this PR is ready.  Items
keep getting added in and those have not
been reviewed...

**Pull request closed without merging**